examples/website/src/shared/types.ts

```diff
@@ -1,10 +1,5 @@
 import {Node} from 'slate'
-import {
-  CommentAuthorType,
-  CommentItemType,
-  RichTextTextNode,
-  User
-} from '../../../../packages/api/lib'
+import {CommentAuthorType, CommentItemType, User} from '../../../../packages/api/lib'
```

Comment:
This import looks very wrong? Did you add this?

Reply:
Done